src/ng-select/lib/ng-select.component.html

```diff
@@ -4,47 +4,59 @@
-          </span>
-        </ng-template>
+  (focus)="onWidgetFocus()">
 
-        <ng-template [ngTemplateOutlet]="labelTemplate || defaultLabelTemplate" [ngTemplateOutletContext]="{ item: item.value, clear: clearItem, label: item.label }"> </ng-template>
-      </div>
-    </ng-container>
+  <div class="ng-value-container" [attr.role]="multiple ? 'listbox' : null" [attr.aria-labelledby]="labelledBy">
+    @if (((multiple && !multiLabelTemplate) || !searchable) && selectedItems.length > 0) {
+        @for (item of selectedItems; track trackByOption($index, item)) {
+            <div
+              #tag
+              class="ng-value"
+              [class.ng-value-disabled]="item.disabled"
+              id="{{ item.htmlId }}_selected"
+              tabindex="-1"
+              role="option"
+              [attr.aria-disabled]="item.disabled"
+              aria-selected="true"
+              (blur)="onTagBlur($event)">
+                <ng-template #defaultLabelTemplate>
+                  <span class="ng-value-label" [ngItemLabel]="item.label" [escape]="escapeHTML"></span>
+                  <span class="ng-value-icon left" (click)="unselect(item)" aria-hidden="true">
+                    <svg aria-hidden="true" focusable="false" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 320 512"><path fill="currentColor" d="M207.6 256l107.72-107.72c6.23-6.23 6.23-16.34 0-22.58l-25.03-25.03c-6.23-6.23-16.34-6.23-22.58 0L160 208.4 52.28 100.68c-6.23-6.23-16.34-6.23-22.58 0L4.68 125.7c-6.23 6.23-6.23 16.34 0 22.58L112.4 256 4.68 363.72c-6.23 6.23-6.23 16.34 0 22.58l25.03 25.03c6.23 6.23 16.34 6.23 22.58 0L160 303.6l107.72 107.72c6.23 6.23 16.34 6.23 22.58 0l25.03-25.03c6.23-6.23 6.23-16.34 0-22.58L207.6 256z" /></svg>
+                  </span>
+                </ng-template>
+                <ng-template
+                    [ngTemplateOutlet]="labelTemplate || defaultLabelTemplate"
+                    [ngTemplateOutletContext]="{ item: item.value, clear: clearItem, label: item.label }">
+                </ng-template>
+            </div>
+        }
+    }
+
+    @if (multiple && multiLabelTemplate && selectedValues.length > 0) {
+      <ng-template [ngTemplateOutlet]="multiLabelTemplate" [ngTemplateOutletContext]="{ items: selectedValues, clear: clearItem }">
+      </ng-template>
+    }
 
-    <ng-template *ngIf="multiple && multiLabelTemplate && selectedValues.length > 0" [ngTemplateOutlet]="multiLabelTemplate" [ngTemplateOutletContext]="{ items: selectedValues, clear: clearItem }"> </ng-template>
+    <div
+      class="ng-input"
+      role="combobox"
+      [attr.aria-expanded]="isOpen"
+      [attr.aria-owns]="isOpen ? dropdownId : null"
+      aria-haspopup="listbox">

```

Comment:
Since this div is non-interactive these are technically on the wrong elements and should be on the input itself - this was part of our original a11y work so I'm a bit wary to see them added back in. I trust the testing, but remember needing to shift a lot of this due to axe issues.

Reply:
@geebru you're right, this was wrongfully placed here. I've removed it